generator.yml

```diff
@@ -48,6 +51,7 @@ architectures:
 triples:
   - &arm-triple         {CROSS_COMPILE: arm-linux-gnueabi-,    ARCH: *arm-arch}
   - &arm64-triple       {CROSS_COMPILE: aarch64-linux-gnu-,    ARCH: *arm64-arch}
+  - &hexagon-triple     {CROSS_COMPILE: hexagon-linux-gnu-,    ARCH: *hexagon-arch}
```

Comment:
It may not matter much at this point but I would recommend `hexagon-unknown-linux-musl-`.

Reply:
done